test/plugin/base_test.rb

```diff
@@ -1628,18 +1628,11 @@ def http_request_message_with_absent_referer
   # The conversions from user input to output.
   def latency_conversion
     {
-      '1.5 m' => { 'seconds' => 90, 'nanos' => 0 },
-      '1 min 32 s' => { 'seconds' => 92, 'nanos' => 0 },
-      '1 min' => { 'seconds' => 60, 'nanos' => 0 },
       '32 s' => { 'seconds' => 32, 'nanos' => 0 },
       '32s' => { 'seconds' => 32, 'nanos' => 0 },
-      '32 second' => { 'seconds' => 32, 'nanos' => 0 },
-      '32 seconds' => { 'seconds' => 32, 'nanos' => 0 },
-      '32 sec' => { 'seconds' => 32, 'nanos' => 0 },
-      '32 secs' => { 'seconds' => 32, 'nanos' => 0 },
       '0.32s' => { 'seconds' => 0, 'nanos' => 320_000_000 },
-      '123' => { 'seconds' => 123, 'nanos' => 0 },
-      '1.3442' => { 'seconds' => 1, 'nanos' => 344_200_000 }
+      ' 123 s ' => { 'seconds' => 123, 'nanos' => 0 },
+      '1.3442 s' => { 'seconds' => 1, 'nanos' => 344_200_000 }
```

Comment:
Can we make the tabs explicit, please? I.e., use `"\t\t123\t\ts\t\t"` instead of embedding tabs inline.
Also use `\r`, `\n`, `\v`, `\f`...